BrowserKit/Sources/MenuKit/MenuTableView.swift

```diff
@@ -125,7 +125,13 @@ final class MenuTableView: UIView,
     }
 
     func reloadTableView(with data: [MenuSection]) {
-        menuData = data
+        if let firstSection = data.first, firstSection.isTopTabsSection {
```

Comment:
nit: Can you add documentation explaining that if the first section `isTopTabsSection` will be handled in the collectionView and will be ignored by the tableView 🙏 . Can be added in next PRs